client/src/components/mapboxMap.module.scss

```diff
@@ -0,0 +1,19 @@
+.mapContainer {
+  height: 676px;
+  margin-bottom: 29px;
+  max-width: revert;
+  margin-top: 50px;
+}
+
+.sidebar {
+  background-color: rgba(35, 55, 75, 0.9);
```

Comment:
Might want to use color variables if we're going to be tweaking a lot. Not sure it's worth that though.

Reply:
Good callout!